src/App.js

```diff
@@ -55,7 +55,7 @@ function App() {
   const classes = useStyles();
 
   //request
-  const [request, setRequest] = useState("");
+  const [request, setRequest] = useState(false);
   const [loading, setLoading] = useState(false);
```

Comment:
I'll just check what this does in relation to the other things that rely on the setRequest state, might need to do a few tweaks.